cnv_germline_case_scattered_workflow.wdl

```diff
@@ -0,0 +1,253 @@
+#
+# A wrapper for the gCNV case workflow intended for lowering computing cost by making it feasible to use 
+# preemptible cloud instances with low memory requirements. CPU, memory and disk requirements can be 
+# lowered for GermlineCNVCaller and DetermineGermlineContigPloidy tasks. 
+# 
+#
+# - Example invocation:
+#
+#       java -jar cromwell.jar run cnv_germline_case_scattered_workflow.wdl -i my_parameters.json
+#
+####################
+
+#import "cnv_germline_case_workflow.wdl" as GermlineCNVCaseWorkflow
```

Comment:
Is this worth keeping?

Reply:
Just tested this today, looks like Dockstore and Terra are able to handle relative paths for imports from git. So git raw file urls are no longer required. Neat!
The files have been updated to remove the urls.